pkg/debuginfo/debuginfod.go

```diff
@@ -0,0 +1,44 @@
+package debuginfo
+
+import (
+	"io"
+	"net/http"
+	"path"
+)
+
+var publicServer = "https://debuginfod.systemtap.org"
```

Comment:
Since the flag defines the default, this seems unused. I think we can safely remove it.